### ISSUE TITLE
v2.0.5. Not the way described in documentation

### ISSUE THREAD

**Issue by oRastor** [deployphp/deployer#202]
At page http://deployer.org/docs/servers i can see information:
>This function gets 3 parameters server(server_name, host, port) and return a Deployer\Server\Configuration object which contains the server configuration.

But really function ```server``` now return ```Deployer\Server\Builder``` instance. Is it right?

Before i used v2.0.2.

**Comment by elfet** [comment]
Docs is outdated, sorry. Now working on rewriting it. 
Current version v3 is very differs from 2.x.

**Comment by elfet** [comment]
Try this:
```json
{
    "require": {
        "deployer/deployer": "2.0.5"
    },
    "provide": {
        "ext-ssh2": "*"
    },
    "minimum-stability": "dev"
}
```

**Comment by oRastor** [comment]
I've updated deployer with self-update command. When I did this deployer has downloaded 2.0.5 version. However this version (2.0.5) was 3.0+ really. That's why old recepies weren't working. I've built new deployer.phar from 2.x branch.

**Comment by oRastor** [comment]
Problem not resolved if extension ssh2 enabled.
https://github.com/deployphp/deployer/issues/201

**Comment by elfet** [comment]
Fixed in #201
